runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -298,7 +298,7 @@ private boolean isFavoriteConfig(int world)
 	private void clearFavoriteConfig(int world)
 	{
 		configManager.unsetConfiguration(WorldHopperConfig.GROUP, "favorite_" + world);
-		panel.resetAllFavoriteMenus();
+		updateList();
```

Comment:
Is this actually necessary? `removeFromFavorites` looks like it is already correctly updating the favorite menu so I don't see why this is required.

Reply:
I've tested this and it seems unnecesary